components/side-bar.tsx

```diff
@@ -67,23 +28,20 @@ const SideBar: FC<SideBarProps> = ({ inSideDrawer = false }) => {
         className={`${!inSideDrawer && "hidden"} ${
           !inSideDrawer && "lg:block"
         } h-full bg-themeGray lg:min-h-screen  ${
-          !inSideDrawer && "md:w-[239px] md:min-w-[239px]"
-        } ${!inSideDrawer && "lg:w-[239px] lg:min-w-[239px]"}`}
+          !inSideDrawer && "md:w-[15rem] md:min-w-[15rem]"
+        } ${!inSideDrawer && "lg:w-[15rem] lg:min-w-[15rem]"}`}
```

Comment:
you are using same width for screens after `md`, no need to add `${!inSideDrawer && "lg:w-[15rem] lg:min-w-[15rem]"}`
`min-w` and `width` are some, you can remove `min-w`